website/pages/docs/index.mdx

```diff
@@ -7,12 +7,12 @@ GraphQL.JS is the reference implementation to the [GraphQL Specification](https:
 while closely following the Specification.
 
 You can build GraphQL servers, clients, and tools with this library, it's designed so you can choose which parts you use, for example, you can build your own parser
-and use the execution/validation from the library. There also a lot of useful utilities for schema-diffing, working with arguments and [many more](./utilities.mdx).
```

Comment:
this link is 404, so I removed it cc @JoviDeCroock

Reply:
https://www.graphql-js.org/api-v16/utilities/ exists?

Reply:
yeah, should I use this link instead?

Reply:
Is there an evergreen link that can be used instead? like `graphql-js.org/latest/utilities`?

Reply:
But if not, then yeah might as well I guess :+1: